core/src/main/scala/fs2/Async.scala

```diff
@@ -189,6 +189,13 @@ object Async {
    */
   case class Change[+A](previous: A, now: A)
 
+  object Change {
+    implicit class ChangeSyntax[A](val self:Change[A]) extends AnyVal {
```

Comment:
I am just used to do it that way. Is more less style, and form my understanding the compiler emits same code at the end.